airflow/providers/google/leveldb/operators/leveldb.py

```diff
@@ -0,0 +1,116 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from typing import Callable, List, Optional
+
+from airflow.models import BaseOperator
+from airflow.providers.google.leveldb.hooks.leveldb import LevelDBHook
+from airflow.utils.decorators import apply_defaults
+
+
+class LevelDBOperator(BaseOperator):
+    """
+    Execute command in LevelDB
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:LevelDBOperator`
+    """
+
+    # pylint: disable=too-many-instance-attributes
+    # Twenty is reasonable in this case.
+
+    @apply_defaults
+    def __init__(
+        self,
+        *,
+        command: str,
+        key: bytes,
+        value: bytes = None,
+        keys: List[bytes] = None,
+        values: List[bytes] = None,
```

Comment:
Since these arguments accept bytes, it'll be good to have a validation that checks them and report or convert them to bytes? What do you think?

Reply:
ok, I add

Reply:
Hey @radim94, looks like a made a stupid request. Since these arguments have type hints, I think it's ok not to validate. Please discard the last commit. Apologies.